pkg/k8shandler/status.go

```diff
@@ -91,7 +91,7 @@ func getNodeStatus(name string, status *v1alpha1.ElasticsearchStatus) (int, *v1a
 
 func rolePodStateMap(namespace string, clusterName string) map[v1alpha1.ElasticsearchNodeRole]v1alpha1.PodStateMap {
 
-	baseSelector := fmt.Sprintf("component=%s", clusterName)
+	baseSelector := fmt.Sprintf("component=elasticsearch,cluster-name=%v", clusterName)
```

Comment:
Will we need to change https://github.com/openshift/origin-aggregated-logging/blob/master/hack/testing/util.sh#L17 too?

Reply:
It seems like we need to simplify some of that code. This would work https://github.com/openshift/origin-aggregated-logging/blob/master/hack/testing/util.sh#L22

Not sure how the CRs are being created for those tests (will we have a CR named "elasticsearch-infra"?)

Reply:
This is how we create the CR for CL: https://github.com/openshift/origin-aggregated-logging/blob/master/openshift/ci-operator/build-image/setup-logging-for-e2e.sh#L154

This is the CR we use: https://github.com/openshift/origin-aggregated-logging/blob/master/openshift/ci-operator/build-image/cr.yaml

Reply:
the CLO will create the Elasticsearch CR with the name `elasticsearch` every time. so I don't believe this impacts that as the cr-name and component would have both been `elasticsearch`